crosswalks/MODS.csv

```diff
@@ -0,0 +1,68 @@
+﻿Property,MODS
+codeRepository,relatedItem/[@type='Host']
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,location/url[@displayName='Download URL']
+fileSize,physicalDescription/extent
+installUrl,location/url[@displayName='Install URL']
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,description
+softwareHelp,relatedItem
+softwareRequirements,
+softwareVersion,originInfo/edition
+storageRequirements,
+supportingData,relatedItem[@type='references']
+author,name/role/roleTerm[@type='code']/aut
+citation,relatedItem[@otherType='references']
+contributor,name/role/roleTerm[@type='code']/ctb
+copyrightHolder,accessCondition/rights.holder/name
+copyrightYear,accessCondition/creation/year.creation
+dateCreated,originInfo/dateCreated
+dateModified,originInfo/dateModified
+datePublished,originInfo/dateIssued
+editor,name/role/roleTerm[@type='code']/edt
+encoding,
+fileFormat,typeOfResource
+funder,name/role/roleTerm[@type='code']/fnd
+keywords,subject
+license,accessCondition[@type='use and reproduction']
+producer,name/role/roleTerm[@type='code']/pro
+provider,name/role/roleTerm[@type='code']/prv
+publisher,originInfo/publisher
+sponsor,name/role/roleTerm[@type='code'
+version,originInfo/edition
+isAccessibleForFree,accessCondition[@type='accessible for free']/yes
+isPartOf,part
+hasPart,part
+position,part/[@order='']
+description,abstract
+identifier,identifier
+name,titleInfo/title
+sameAs,location/url[@displayName='Same As']
+url,location/url
+relatedLink,relatedItem/location/url[@displayName='Related Link']
+givenName,name/namePart[@type='given']
+familyName,name/namePart[@type='family']
+email,name/affiliation

```

Comment:
```suggestion
email,
```

That doesn't seem right. `<affiliation>` can contain an email, but it isn't the email itself according to [this spec](https://www.loc.gov/standards/mods/userguide/name.html#affiliation)